src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -53,12 +53,11 @@ namespace OpenMS
   EnzymesDB::EnzymesDB()
   {
     readEnzymesFromFile_("CHEMISTRY/Enzymes.xml");
```

Comment:
I've opened a ticket for the problem ("share" set incorrectly): https://github.com/OpenMS/OpenMS/issues/1479